src/test/java/subway/SectionAcceptanceTest.java

```diff
@@ -0,0 +1,237 @@
+package subway;
+
+import io.restassured.RestAssured;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.jdbc.Sql;
+import subway.commons.ErrorCode;
+
+import java.text.MessageFormat;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import static org.assertj.core.api.Assertions.assertThat;
+
+@DisplayName("구간관련 기능")
+@Sql(scripts = {"StationInsert.sql", "LineInsert.sql", "SectionInsert.sql"})
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT)
+public class SectionAcceptanceTest {
+
+    /**
+     * given: 기존에 노선과 역이 등록되어 있고,
+     * when: 관리자가 구간을 등록하면
+     * then: 구간의 하행 종점역이 바뀐다.
+     */
+    @DirtiesContext
+    @Test
+    @DisplayName("기존의 노선에 구간을 등록한다.")
+    void 구간_등록_테스트() {
+        //given
+        int testingLineNumber = 1;
+        String terminalStationName = "강남역";
+        String upStationId = "5";
+        String downStationId = "6";
+        String distance = "10";
+        Map<String, String> params = new HashMap<>();
+        params.put("upStationId", upStationId);
+        params.put("downStationId", downStationId);
+        params.put("distance", distance);
+
+        //when
+        ExtractableResponse response = postAPIResponse("/lines/" + testingLineNumber + "/sections", params);
```

Comment:
제네릭 타입을 명시해주시면 좋을 것 같습니다.
```suggestion
        ExtractableResponse<Response> response = postAPIResponse("/lines/" + testingLineNumber + "/sections", params);
```